uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-work-patterns.html

```diff
@@ -54,3 +54,73 @@
   </tr>
   </tbody>
 </table>
+
+
+<!-- Modal -->
+<div class="modal fade" id="workPatternModal" tabindex="-1" role="dialog">
```

Comment:
The modal is static at this stage, but even if it's static we should be able to open it by clicking a button in the  work pattern page, instead of having it in the tab's view file.

Please move this to a separate view file and add a simply method to open the modal in the [absence-work-patterns-tab](https://github.com/civicrm/civihr/blob/6e2eda4fce2234f19fcf5e59f470302922293977/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-work-patterns.js)